app/views/omni_nav/_off_canvas_nav.html.erb

```diff
@@ -1,173 +0,0 @@
-
```

Comment:
I renamed the `omni_nav` views directory `omni_nav_v2` to be consistent with the stylesheet layout.